### ISSUE TITLE
Impossible to display ApplicationInsight/CustomMetrics

### ISSUE THREAD

**Issue by NikokoFR** [grafana/azure-monitor-datasource#24]
Hello,

When i'm selecting a CustomMetrics from the combo, no values are returned.
![grafana](https://user-images.githubusercontent.com/34578718/34049705-b5410a1c-e1b8-11e7-8bc7-bc746e4fcc83.JPG)

This is the answer i get in the Query Inspector 

```
xhrStatus:"complete"
request:Object
method:"GET"
url:"api/datasources/proxy/1/appinsights/beta/apps/<blablabla>/metrics/customMetrics/\System\System Up Time?timespan=2017-12-15T15:19:40Z/2017-12-15T15:49:40Z&aggregation=avg&interval=PT10M"
response:Object
value:Object
start:"2017-12-15T15:19:40.000Z"
end:"2017-12-15T15:49:40.000Z"
interval:"PT10M"
segments:Array[0]
```

I checked in my interface, there are values available
![uptime](https://user-images.githubusercontent.com/34578718/34049666-952a25ce-e1b8-11e7-8c9b-27a358ec3597.JPG)

Not sure this is a bug but i cant figure out what is wrong here

Thank you !

**Comment by NikokoFR** [comment]
I think i found the issue:
When i'm requesting "System Up Time" is see a problem in the query format.
The backslashes are replaced with simple slashes:
/metrics/customMetrics/**/**System**/**System Up Time?timespan=2017-12-11T16:24:52Z/2017-12-18T16:24:52Z&aggregation=avg&interval=PT10M

By changing the URL manually, i can make it work with
/metrics/customMetrics/**\**System**\**System Up Time?timespan=2017-12-11T16:24:52Z/2017-12-18T16:24:52Z&aggregation=avg&interval=PT10M

**Comment by daniellee** [comment]
closed by #52
